web-client/src/presenter/actions/clearIdleTimerAction.ts

```diff
@@ -0,0 +1,9 @@
+import { state } from '@web-client/presenter/app.cerebral';
+
+export const clearIdleTimerAction = ({ store }: ActionProps) => {
+  store.unset(state.lastIdleAction);
+  store.set(state.idleLogoutState, {
+    logoutAt: undefined,
+    state: 'INITIAL',
+  });
+};
```

Comment:
We were never clearing this on sign out. This led to a minor (and humorous!) bug in which a user who waited on the screen for 55 minutes after signing out would see the "Are you still there?" modal.